1.8/usage/service-discovery/haproxy-adminrouter.md

```diff
@@ -0,0 +1,107 @@
+---
+post_title: Configuring HAProxy in Front of Admin Router
+nav_title: HAProxy in Front of Admin Router
+menu_order: 400
+---
+
+You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter).
```

Comment:
> let me know your thoughts

Well, this is IMO low priority and not risk-free. So, let's save this for later.